.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E203, E221, E241, E266, W291, E302, F403, F405, E501, W503, E741
```

Comment:
@tzaffi Minor:  Unless opposed, can you refactor to one entry per line?

In my experience, having a single entry per line reduces the chance of mistake during PR review as we update entries.

Reply:
👍 

Looking more closely at the [flake8's config definition](https://flake8.pycqa.org/en/latest/user/configuration.html) I believe your suggestion will work and I'll try it